### DISCUSSION TITLE
Support from ftp

### DISCUSSION THREAD

**Discussion by siamakabi** [appwrite/appwrite#5902] [Ideas]
Is there a way to upload and store files via ftp on a separate server?
I did not find anything

I hope this feature will be added

**Comment by stnguyen90** [comment]
Unfortunately not. It must go through Appwrite so Appwrite can create the related entries in the database.

**Reply by siamakabi** [reply]
I think using S-3 like [pocketbase](https://pocketbase.io/) is a valuable method
I hope this method will be used in appwrite in the future
